components/statuspage/sources/incident-updated/incident-updated.mjs

```diff
@@ -0,0 +1,37 @@
+import common from "../common/common.mjs";
+
+export default {
+  ...common,
+  name: "Incident Updated (Instant)",
+  version: "0.0.1",
+  key: "statuspage-incident-updated",
+  description: "Emit new event on each updated incident.",
+  type: "source",
+  dedupe: "unique",
+  methods: {
+    ...common.methods,
+    async deploy() {
+      const incidents = await this.statuspage.getIncidents({
+        pageId: this.pageId,
+        params: {
+          limit: 20,
+        },
+      });
+
+      incidents.reverse().forEach(this.emitEvent);
+    },
+    emitEvent(event) {
+      const data = event?.incident ?? event;
+
+      if (data.incident_updates.length <= 1) {
```

Comment:
Is there a reason we skip when there is only one?

Reply:
Yes, because if just have one it's a new incident, not an updated.